index.html

```diff
@@ -474,7 +473,96 @@ <h2>Well Known URIs</h2>
     </p>
 
 
-
+    <section>
+      <h2>OpenID Connect</h2>
+      <p>
+        OIDC uses <a data-cite="RFC5785#section-3">Well-Known Uniform Resource Identifiers (URIs)</a> 
```

Comment:
The OpenID Foundation always spells out "OpenID Connect" in full.

Reply:
https://github.com/w3c/vc-jose-cose/pull/122/files#r1264091221